packages/web/components/Landing/LandingConnectButton.tsx

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable no-nested-ternary */
+import {
+  Box,
+  Button,
+  Icon,
+  Spinner,
+  Text,
+  Tooltip,
+  useBreakpointValue,
+  useToast,
+  VStack,
+} from '@metafam/ds';
+import { MetaLink } from 'components/Link';
+import { useWeb3 } from 'lib/hooks';
+import { useEffect, useState } from 'react';
+import { GoSignIn, GoSignOut } from 'react-icons/go';
+import { shortenAddress } from 'utils/stringHelpers';
+
+export const LandingConnectButton = ({ isIconStyle = false, ...props }) => {
+  const {
+    connect,
+    disconnect,
+    connected,
+    connecting,
+    address,
+    chainId,
+    provider,
+  } = useWeb3();
+  const [wrongNetwork, setWrongNetwork] = useState(false);
+  const spinnerSize = useBreakpointValue({ base: 'sm', '2xl': 'md' });
+  const toast = useToast();
+
+  useEffect(() => {
+    if (connected && address && provider) {
+      const networkName = provider.network.name;
+      const shortAddress = shortenAddress(address);
+      console.log('Connected to', networkName, 'with address', shortAddress);
+
+      if (chainId !== '0x89') {
+        setWrongNetwork(true);
+        toast({
+          title: `Wallet Connection`,
+          description: (
+            <VStack spacing={2} justifyItems="start">
+              <Text>
+                <Box as="strong" textTransform="capitalize">
+                  {networkName ?? ''} ({chainId})
+                </Box>{' '}
+                network is not supported. This area of the website requires{' '}
+                <strong>Polygon (id: 0x89)</strong> network. Please switch
+                networks.
+              </Text>
+              <Text>
+                You can visit{' '}
+                <MetaLink href="https://chainlist.org/chain/137" isExternal>
```

Comment:
There is a `web3/components/SwitchNetworkButton.tsx` that you can use to help users automatically add this network to their metamask.

We could modify that component to have a link to `chainlist.org` as you've done here.

Reply:
OK, I'll look at modifying the `SwitchNetworkButton` component to include the Polygon network and a chainlist link (it's a very useful site for folks to know about).

Reply:
I have been trying to add the `<SwitchNetworkButton chainId="0x89" />` to the toast here but it doesn't render anything. Logging the values to the console shows `isMetaMask = false`  when I am using MM to connect.